# motion type: Go, Stop
string motion

# movement type: Stepping, Viewing
string movement

# linear velocity
float32 x_velocity
float32 y_velocity

# angular rate
float32 rate

# viewing params
float32 roll
float32 pitch
float32 yaw

# robot height
float32 z

# Step adjust
float32 faster
float32 slower